src/block/progress-circle/edit.js

```diff
@@ -150,6 +152,9 @@ const Edit = props => {
 						) }
 					</div>
 				</div>
+				<style>
+					{ `.editor-styles-wrapper .stk-${ props.attributes.uniqueId } .stk-progress-circle { --progress-value:${ derivedValue } }` }
+				</style>
```

Comment:
```suggestion
				{ /* Add our progress style here because we're adjusting the value using a hook */ }
				<style>
					{ `.editor-styles-wrapper .stk-${ props.attributes.uniqueId } .stk-progress-circle { --progress-value:${ derivedValue } }` }
				</style>
```